src/cljs/rems/actions/invite_decider_reviewer.cljs

```diff
@@ -76,12 +76,12 @@
      :decider decider-form-id
      :reviewer reviewer-form-id)
    (case role
-     :decider (text :t.actions/invite-decider)
-     :reviewer (text :t.actions/invite-reviewer))
+     :decider (str (text :t.actions/request-decision-menu) " " (text :t.actions/invite-decider-suffix))
+     :reviewer (str (text :t.actions/request-review-menu) " " (text :t.actions/invite-decider-suffix)))
```

Comment:
Also here it would be better to not concatrnate strings but use templates as they are from tempura or with format parameters.

Reply:
Would you prefer (text :t.actions/request-decision-menu) had a parameter at the very end? That feels like overkill when we can just catenate. The localizations this UI needs are

- The action buttons: "Request review", "Request decision"
- The dropdown elements: "... from user", "... via email"
- The form titles "Request review from user", "Request review by email" etc.
- The submit buttons: "Request review", "Request decision"

I feel the more verbose form titles are important so the user can see that his selection was honoured. But if the form title were just "Request review", we wouldn't need string operations for translations.

Reply:
The point is just leaving these details open, and to be within the translation system. I think there is not much advantage in trying to DRY the translations. And clever use of `text-format`, especially multiple rounds of translations, does not feel like right too. I think the simplest, and within the translation system still, would be to duplicate the translations themselves.

Maybe one option is to make the `...` operation a helper in the text namespace so that we could mostly avoid the CSS hack and the string concatenation and leave it to our translation system from where we can fix it easily if we need to.

Reply:
fair enough

Reply:
fixed

Reply:
I think it looks pretty good like this :+1: